ChemistryLib/PhreeqcInterfaceData/AqueousSolution.h

```diff
@@ -0,0 +1,71 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2019, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#pragma once
+
+#include <boost/optional/optional.hpp>
+#include <fstream>
```

Comment:
Would it be possible to use `<iosfwd>` in the header, and then you might have to include the `<fstream>` in the implementation.

Reply:
Redo as suggested.